gift_compassion/models/sponsorship_gift.py

```diff
@@ -109,9 +111,9 @@ class SponsorshipGift(models.Model):
     @api.model
     def get_gift_type_selection(self):
         return [
-            ('Project Gift', _('Project Gift')),
-            ('Family Gift', _('Family Gift')),
-            ('Beneficiary Gift', _('Beneficiary Gift')),
+            ('Project', _('Project')),
```

Comment:
Only change the right part of the tuples. The left part is the value in the database and is used as well for the mapping when receiving messages, so this will break the code if we modify it.